### ISSUE TITLE
Crashes after quit Application in macOS Sierra 10.12.3

### ISSUE THREAD

**Issue by dudisbr** [pgmodeler/pgmodeler#903]
The application crashes everytime you close it on macOS Sierra 10.12.3

Stack trace as follows:

```
** pgModeler crashed after receive signal: 11 **

Date/Time: 2017-02-03 14:45:03 
Version: 0.9.0-alpha 
Build: 20161017 
Compilation Qt version: 5.5.1
Running Qt version: 5.5.1

[29] 0   pgmodeler                           0x00000001079c7e20 _Z17startCrashHandleri + 64
[28] 1   libsystem_platform.dylib            0x00007fffc8cbabba _sigtramp + 26
[27] 2   ???                                 0x0000000000000000 0x0 + 0
[26] 3   libFosl_dynamic.dylib               0x00007fffc7259f75 libFosl_dynamic.dylib + 16245
[25] 4   libFosl_dynamic.dylib               0x00007fffc725a9ac libFosl_dynamic.dylib + 18860
[24] 5   libFosl_dynamic.dylib               0x00007fffc725abcc libFosl_dynamic.dylib + 19404
[23] 6   libFosl_dynamic.dylib               0x00007fffc7324ee3 fosl_astPrintReachable + 796387
[22] 7   libFosl_dynamic.dylib               0x00007fffc732a52e fosl_filter_createGraph + 94
[21] 8   CoreImage                           0x00007fffb4069f73 _ZN2CI11FoslAdapterC2EPKNS_7ContextEPKNS_11ProgramNodeEPNS_14SerialIntArrayEPNS_17SerialStringArrayEPNS_20SerialObjectPtrArrayE + 311
[20] 9   CoreImage                           0x00007fffb4069437 _ZN2CI13ShaderBuilderC1EPKNS_7ContextEPKNS_4NodeEPKNS_11ProgramNodeEPNS_14SerialIntArrayEPNS_17SerialStringArrayEPNS_20SerialObjectPtrArrayE + 41
[19] 10  CoreImage                           0x00007fffb403fdf5 _ZN2CI11ProgramNode28create_program_and_argumentsEPNS_7ContextEPKc + 1467
[18] 11  CoreImage                           0x00007fffb403ef77 _ZN2CI11ProgramNode19build_program_graphEPNS_4NodeEPNS_7ContextERNS_13NodeToNodeMapEb + 1487
[17] 12  CoreImage                           0x00007fffb3f9613b _ZN2CI4Node13program_graphEPNS_7ContextERNS_13NodeToNodeMapEb + 63
[16] 13  CoreImage                           0x00007fffb3fa51c5 _ZN2CIL32make_program_graph_if_renderableEPNS_7ContextEPKcPNS_4NodeERK6CGRectRKNS_12swizzle_infoEPcm + 380
[15] 14  CoreImage                           0x00007fffb3fa473b _ZN2CI14create_cgimageEPNS_7ContextEPNS_5ImageE6CGRectP12CGColorSpaceNS_11PixelFormatEbm + 2591
[14] 15  CoreImage                           0x00007fffb3f87b50 -[CIContext(Internal) _createCGImage:fromRect:format:colorSpace:deferred:textureLimit:] + 1057
[13] 16  CoreImage                           0x00007fffb3f86957 -[CIContext createCGImage:fromRect:format:colorSpace:] + 68
[12] 17  CoreImage                           0x00007fffb3f867f6 -[CIContext createCGImage:fromRect:] + 299
[11] 18  AppKit                              0x00007fffb1437dbe -[NSBitmapImageRep initWithCIImage:] + 674
[10] 19  AppKit                              0x00007fffb14c795d -[NSColorPickerWheelView _compositeImage] + 511
[9] 20  AppKit                              0x00007fffb14c7f03 -[NSColorPickerWheelView viewDidChangeBackingProperties] + 90
[8] 21  AppKit                              0x00007fffb0ffde8b _NSViewHierarchyDidChangeBackingProperties + 433
[7] 22  AppKit                              0x00007fffb0fbffbe -[NSView addSubview:] + 1242
[6] 23  AppKit                              0x00007fffb0ffb7cb -[NSFrameView addSubview:] + 45
[5] 24  AppKit                              0x00007fffb0ffb78f -[NSThemeFrame addSubview:] + 383
[4] 25  AppKit                              0x00007fffb0ffb3c8 -[NSView addSubview:positioned:relativeTo:] + 217
[3] 26  AppKit                              0x00007fffb1811011 -[NSThemeFrame addSubview:positioned:relativeTo:] + 43
[2] 27  AppKit                              0x00007fffb0ffb2e2 -[NSThemeFrame _addKnownSubview:positioned:relativeTo:] + 38
[1] 28  AppKit                              0x00007fffb10233a7 -[NSWindow setContentView:] + 647
[0] 29  libqcocoa.dylib                     0x000000010af68113 qt_plugin_instance + 202179
```

**Comment by dudisbr** [comment]
The Application Crash still happens on v.0.9.0-apha1

```
** pgModeler crashed after receive signal: 11 **

Date/Time: 2017-02-10 21:08:39 
Version: 0.9.0-alpha1 
Build: 20170207 
Compilation Qt version: 5.6.2
Running Qt version: 5.6.2

[29] 0   pgmodeler                           0x00000001062ebc00 pgmodeler + 23552
[28] 1   libsystem_platform.dylib            0x00007fffbbd25bba _sigtramp + 26
[27] 2   ???                                 0x0000000000000000 0x0 + 0
[26] 3   libFosl_dynamic.dylib               0x00007fffba2c4f75 libFosl_dynamic.dylib + 16245
[25] 4   libFosl_dynamic.dylib               0x00007fffba2c59ac libFosl_dynamic.dylib + 18860
[24] 5   libFosl_dynamic.dylib               0x00007fffba2c5bcc libFosl_dynamic.dylib + 19404
[23] 6   libFosl_dynamic.dylib               0x00007fffba38fee3 fosl_astPrintReachable + 796387
[22] 7   libFosl_dynamic.dylib               0x00007fffba39552e fosl_filter_createGraph + 94
[21] 8   CoreImage                           0x00007fffa70d4f73 _ZN2CI11FoslAdapterC2EPKNS_7ContextEPKNS_11ProgramNodeEPNS_14SerialIntArrayEPNS_17SerialStringArrayEPNS_20SerialObjectPtrArrayE + 311
[20] 9   CoreImage                           0x00007fffa70d4437 _ZN2CI13ShaderBuilderC1EPKNS_7ContextEPKNS_4NodeEPKNS_11ProgramNodeEPNS_14SerialIntArrayEPNS_17SerialStringArrayEPNS_20SerialObjectPtrArrayE + 41
[19] 10  CoreImage                           0x00007fffa70aadf5 _ZN2CI11ProgramNode28create_program_and_argumentsEPNS_7ContextEPKc + 1467
[18] 11  CoreImage                           0x00007fffa70a9f77 _ZN2CI11ProgramNode19build_program_graphEPNS_4NodeEPNS_7ContextERNS_13NodeToNodeMapEb + 1487
[17] 12  CoreImage                           0x00007fffa700113b _ZN2CI4Node13program_graphEPNS_7ContextERNS_13NodeToNodeMapEb + 63
[16] 13  CoreImage                           0x00007fffa70101c5 _ZN2CIL32make_program_graph_if_renderableEPNS_7ContextEPKcPNS_4NodeERK6CGRectRKNS_12swizzle_infoEPcm + 380
[15] 14  CoreImage                           0x00007fffa700f73b _ZN2CI14create_cgimageEPNS_7ContextEPNS_5ImageE6CGRectP12CGColorSpaceNS_11PixelFormatEbm + 2591
[14] 15  CoreImage                           0x00007fffa6ff2b50 -[CIContext(Internal) _createCGImage:fromRect:format:colorSpace:deferred:textureLimit:] + 1057
[13] 16  CoreImage                           0x00007fffa6ff1957 -[CIContext createCGImage:fromRect:format:colorSpace:] + 68
[12] 17  CoreImage                           0x00007fffa6ff17f6 -[CIContext createCGImage:fromRect:] + 299
[11] 18  AppKit                              0x00007fffa44a2dbe -[NSBitmapImageRep initWithCIImage:] + 674
[10] 19  AppKit                              0x00007fffa453295d -[NSColorPickerWheelView _compositeImage] + 511
[9] 20  AppKit                              0x00007fffa4532f03 -[NSColorPickerWheelView viewDidChangeBackingProperties] + 90
[8] 21  AppKit                              0x00007fffa4068e8b _NSViewHierarchyDidChangeBackingProperties + 433
[7] 22  AppKit                              0x00007fffa402afbe -[NSView addSubview:] + 1242
[6] 23  AppKit                              0x00007fffa40667cb -[NSFrameView addSubview:] + 45
[5] 24  AppKit                              0x00007fffa406678f -[NSThemeFrame addSubview:] + 383
[4] 25  AppKit                              0x00007fffa40663c8 -[NSView addSubview:positioned:relativeTo:] + 217
[3] 26  AppKit                              0x00007fffa487c011 -[NSThemeFrame addSubview:positioned:relativeTo:] + 43
[2] 27  AppKit                              0x00007fffa40662e2 -[NSThemeFrame _addKnownSubview:positioned:relativeTo:] + 38
[1] 28  AppKit                              0x00007fffa408e3a7 -[NSWindow setContentView:] + 647
[0] 29  libqcocoa.dylib                     0x00000001098f5a33 qt_plugin_instance + 198003
```

**Comment by rkhaotix** [comment]
This is a bug in Qt framework. Waiting the release of patches

https://bugreports.qt.io/browse/QTBUG-56448

Em 10 de fev de 2017 21:16, "dudisbr" <notifications@github.com> escreveu:

**Comment by dudisbr** [comment]
Problem solved in version 0.9.1-beta release 20180122 running on MacOS Sierra 10.12.6.

Thanks a lot!
